docs/release-notes/8.19.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[release-notes-header-8.19.0]]
+== 8.19
+
+[discrete]
+[[release-notes-8.19.0]]
+=== 8.19.0
+
+[discrete]
+[[deprecations-8.19.0]]
+==== Deprecations
+* Removes default quick prompts from the Security AI Assistant ({kibana-pull}225536[#225536]).
+
+
+[discrete]
+[[features-8.19.0]]
+==== New features
+* Adds an option to update the `kibana.alert.workflow_status` field for alerts associated with attack discoveries ({kibana-pull}225029[#225029]).
+* The rule execution gaps functionality is now generally available ({kibana-pull}224657[#224657]).
+* Adds the ability to bulk fill gaps ({kibana-pull}224585[#224585]).
+* Automatic migration is now generally available ({kibana-pull}224544[#224544]).
+* Adds a name field to the automatic migration UI ({kibana-pull}223860[#223860]).
+* Adds the ability to bulk set up and delete alert suppression ({kibana-pull}223090[#223090]).
+* Adds a human-readable incremental ID to cases, making referencing cases easier ({kibana-pull}222874[#222874]).
+* Adds the ability to change rule migration execution settings when re-processing a migration ({kibana-pull}222542[#222542]).
+* Adds `runscript` response action support for Microsoft Defender for Endpoint–enrolled hosts ({kibana-pull}222377[#222377]).
+* Updates automatic migration API schema ({kibana-pull}219597[#219597]).
+* Adds automatic saving of attack discoveries, with search and filter capabilities ({kibana-pull}218906[#218906]).
+* Adds the ability to edit highlighted fields in the alert details flyout ({kibana-pull}216740[#216740]).
+* Adds the XSOAR connector ({kibana-pull}212049[#212049]).
+* Adds a custom script selector for choosing scripts to execute when using the `runscript` response action ({kibana-pull}204965[#204965]).
+
+[discrete]
+[[enhancements-8.19.0]]
+==== Enhancements
+* Updates {elastic-sec} Labs Knowledge Base content ({kibana-pull}227125[#227125]).
+* Displays which fields are customized for prebuilt rules ({kibana-pull}225939[#225939]).
+* Bumps default Gemini model ({kibana-pull}225917[#225917]).
+* Groups vulnerabilities by resource and cloud account using IDs instead of names ({kibana-pull}225492[#225492]).
+* Adds prompt tiles to the Security AI Assistant ({kibana-pull}224981[#224981]).
+* Adds support for collapsible sections in integrations READMEs ({kibana-pull}223916[#223916]).
+* Adds advanced policy settings in {elastic-defend} to enable collection of file origin information for File, Process, and DLL (ImageLoad) events ({kibana-pull}222030[#222030], {kibana-pull}223882[#223882]).
+* Adds the `ecs@mappings` component to the transform destination index template ({kibana-pull}223878[#223878]).
+* Adds the ability to revert prebuilt rules to their base version ({kibana-pull}223301[#223301]).
```

Comment:
Same suggestion as in https://github.com/elastic/docs-content/pull/2149#discussion_r2223152704